doc/lightning-getbalance.7.txt

```diff
@@ -0,0 +1,39 @@
+LIGHTNING-GETBALANCE(7)
+======================
+:doctype: manpage
+
+NAME
+----
+lightning-getbalance - Command showing the total wallet balance.
+
+SYNOPSIS
+--------
+*getbalance* 
+
+DESCRIPTION
+-----------
+The *getbalance* RPC command displays the total funds either on chain or locked on channels.
+
+RETURN VALUE
+------------
+On success an object will be returned with four entries :
+
+- 'on_chain' - The sum of all the wallet utxos, in satoshis.
+
+- 'on_chain_msat' - The sum of all the wallet utxos, in milli satoshis.
+
+- 'on_channels' - The sum of all our satoshis locked in channels, in satoshis.
+
+- 'on_channels_msat' - The sum of all our satoshis locked in channels, in milli satoshis (rounded).
+
+AUTHOR
+------
+Darosior <darosior@protonmail.com> is mainly responsible.
+
+SEE ALSO
+--------
+lightning-listfunds(7), lightning-newaddr(7), lightning-fundchannel(7), lightning-withdraw(7) 
```

Comment:
... also trailing whitespace.

I really hate those when I see a git diff, maybe I am too picky.

Reply:
No you are not, I just did not `git diff̀` the man page :/